cfr.yaml

```diff
@@ -69,35 +69,42 @@
 - id: DOH_ROLLOUT_CONFIRMATION_89
   groups:
     - cfr
-  targeting: "'doh-rollout.enabled'|preferenceValue && !'doh-rollout.disable-heuristics'|preferenceValue && !'doh-rollout.skipHeuristicsCheck'|preferenceValue && !'doh-rollout.doorhanger-decision'|preferenceValue && firefoxVersion >= 89"
-  template: cfr_doorhanger
+  targeting: >-
+    (
+    'doh-rollout.enabled'|preferenceValue ||
+    'doh-rollout.ru.enabled'|preferenceValue ||
+    'doh-rollout.ua.enabled'|preferenceValue
+    ) &&
+    !(
+    'doh-rollout.disable-heuristics'|preferenceValue ||
+    'doh-rollout.skipHeuristicsCheck'|preferenceValue ||
+    'doh-rollout.doorhanger-decision'|preferenceValue
+    ) &&
+    firefoxVersion >= 89
+  template: infobar
   content:
-    skip_address_bar_notifier: true
-    persistent_doorhanger: true
-    anchor_id: PanelUI-menu-button
-    layout: icon_and_message
-    learn_more: dns-over-https
+    priority: 3
+    type: global
     text:
       string_id: cfr-doorhanger-doh-body
     buttons:
-      secondary:
-        - label:
-            string_id: cfr-doorhanger-doh-secondary-button
-          action:
-            type: DISABLE_DOH
-      primary:
-        label:
+      - label:
           string_id: cfr-doorhanger-doh-primary-button-2
         action:
           type: ACCEPT_DOH
+        primary: true
+      - label:
+          string_id: cfr-doorhanger-doh-secondary-button
+        action:
+          type: DISABLE_DOH
+      - label:
+          string_id: notification-learnmore-default-label
+        supportPage: dns-over-https
+        # Use no callback/action so telemetry doesn't mix link and secondary
```

Comment:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1760231

Reply:
Bug https://bugzilla.mozilla.org/show_bug.cgi?id=1760231 mentions  click of learn more closes the infobar, is that desired or fix of 1760231 will keep infobar open

Reply:
`callback: null` makes our infobar implementation not close (normally infobar learn more don't close either I believe, but because we end up setting a callback that doesn't return true, it closes).